### ISSUE TITLE
Essentials Sign Control not working for any of the available signs in 1.14

### ISSUE THREAD

**Issue by inspironwins** [EssentialsX/Essentials#2564]
### Information

**Full output of `/ess version`**:

```
Server version: 1.14.1-R0.1-SNAPSHOT git-Spigot-03bd4b0-17da342 (MC: 1.14.1)
EssentialsX version: 2.16.1.156
EssentialsXProtect version: 2.16.1.156
EssentialsXSpawn version: 2.16.1.156
EssentialsXChat version: 2.16.1.156
EssentialsXAntiBuild version: 2.16.1.156
Vault is not installed. Chat and permissions may not work.
You are running an unsupported server version!
```

### Details

**Description**  
None of the signs that are available in minecraft 1.14 or later do not work with essentials sign control. I have tested this with all the different sign types. I have enabled all the signs in the essentials config. Yes they change color when placed.

**Steps to reproduce**  
1. Set a warp somewhere
2. Make a warp sign to that warp
3. Right click on it

**Expected behavior**  
For the sign to take an action.

**How to fix**  
The fix for this is inside of MaterialUtil.java starting on line 58.
This sign list needs to include the list of signs available inside minecraft 1.14.1.
Replace line 58 with this:
```
        SIGNS = EnumUtil.getAllMatching(Material.class, "SIGN", "SIGN_POST", "WALL_SIGN", "ACACIA_SIGN",
            "ACACIA_WALL_SIGN", "BIRCH_SIGN", "BIRCH_WALL_SIGN", "DARK_OAK_SIGN", "DARK_OAK_WALL_SIGN",
            "JUNGLE_SIGN", "JUNGLE_WALL_SIGN", "LEGACY_SIGN", "LEGACY_SIGN_POST", "LEGACY_WALL_SIGN",
            "OAK_SIGN", "OAK_WALL_SIGN", "SPRUCE_SIGN", "SPRUCE_WALL_SIGN");
```
I have compiled and tested this, and it does work.

Thanks for the hard work on a great piece of software.

**Screenshots**  
https://i.imgur.com/4HQ2t4x.png

**Comment by mdcfe** [comment]
Duplicate of #2514. This was fixed over two weeks ago in the 1.14 branch and experimental builds - you can download experimental builds from the [build server](https://ci.ender.zone/job/EssentialsX-Experimental).

**Comment by inspironwins** [comment]
Oh I did not see that branch, or the experimental repo. Thanks!
